### DISCUSSION TITLE
Domain Joinning at run time for LTSP FAT CLIENTS

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#793] [General] [answered]
In past LTSP executing from lts.conf a script for joinning domain worked fine.

In new ltsp. conf I tested how to join an AD (Linux or Windows) Domain.
- Tried with `POST_INIT_xx`, but joinning is not working.
- Tried at `rc3.d `time (Modified image with an S99join-to-domain script) and WORKS!! (I don't know if it's really in rc3.d time strictily when needed but works)

Script is like that:
```
echo nameserver <AD_LINUX_IP> > /etc/resolv.conf
echo <domain_join_password>|realm join <AD_LINUX_IP>
echo ad_gpo_access_control = disabled >> /etc/sssd/sssd.conf
```

So may be a new `POST_RC3_XX` for commands that require a more advanced time to execute?

**Comment by alkisg** [comment, marked as answer]
`man ltsp.conf` and read about POST_SERVICE_x.

**Reply by Soylor** [reply]
It's perfect.

With POST_SERVICE_X works fine.

Thank you.
